src/core/connection.h

```diff
@@ -35,6 +35,7 @@ typedef union QUIC_CONNECTION_STATE {
         BOOLEAN Connected       : 1;    // Handshake completed.
         BOOLEAN ClosedLocally   : 1;    // Locally closed.
         BOOLEAN ClosedRemotely  : 1;    // Remotely closed.
+        BOOLEAN ClosedSilently  : 1;    // Closed with the QUIC_CLOSE_SILENT flag - go directly to draining state.
```

Comment:
The "go directly to draining state" seems like a possibly misplaced comment - should it go into `QuicConnTryClose` instead?